src/components/MonitorLog/PredatorsTab.jsx

```diff
@@ -55,14 +57,31 @@ PredatorField.propTypes = {
   isDisabled: PropTypes.bool.isRequired,
 };
 
-const PredatorsTab = ({ showHeader, isDisabled, predators }) => {
+const PredatorsTab = ({ showHeader, isDisabled, isTemplate, predators }) => {
```

Comment:
⚠️ **[eslint]** <[no-unused-vars](https://eslint.org/docs/rules/no-unused-vars)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>'showHeader' is defined but never used.